src/test/java/org/folio/circulation/domain/ConfigurationServiceTest.java

```diff
@@ -62,6 +64,39 @@ public void testEmptyJson() {
     assertEquals(DateTimeZone.UTC, service.findDateTimeZone(jsonObject));
   }
 
+  @Test
+  public void shouldCorrectParseIfCheckoutTimeoutDurationIsInteger() {
+    JsonObject jsonConfig = new JsonObject()
+      .put("value", getJsonConfigWithCheckoutTimeoutDurationAsInteger());
+    List<Configuration> records = Collections.singletonList(new Configuration(jsonConfig));
+
+    Integer expectedSessionTimeout = service.findSessionTimeout(records);
+
+    assertEquals(expectedSessionTimeout, new Integer(1));
+  }
+
+  @Test
+  public void shouldCorrectParseIfCheckoutTimeoutDurationIsString() {
+    JsonObject jsonConfig = new JsonObject()
+      .put(VALUE, getJsonConfigWithCheckoutTimeoutDurationAsString());
```

Comment:
```suggestion
      .put(VALUE, getJsonConfigWithCheckoutTimeoutDurationAsString(1));
```

For me, it is more readable to pass the timeout value as parameter, or use the same duration for all the cases.

Reply:
Changed